post.go

```diff
@@ -2,18 +2,34 @@ package fbintegration
 
 import (
 	"fmt"
+	"reflect"
 
 	facebookLib "github.com/huandu/facebook"
 )
 
 type (
 	// Post comment pending
 	Post struct {
-		ID                string `facebook:"id"        json:"post_id"`
-		ObjectID          string `facebook:"object_id" json:"object_id"`
-		Insights          *facebookLib.Result
-		ReactionBreakdown []*facebookLib.Result
-		TotalReactions    *facebookLib.Result
+		ID                 string `facebook:"id"        json:"post_id"`
+		ObjectID           string `facebook:"object_id" json:"object_id"`
+		Insights           *facebookLib.Result
+		ReactionBreakdown  []*facebookLib.Result
+		TotalReactions     *facebookLib.Result
+		Name               string         `json:"name"`
```

Comment:
Might be worth nesting these into a struct that sits under `Insights` and the store the insight data in `Insights.Data`

🏄